docs/contribution-guide.md

```diff
@@ -12,11 +12,31 @@ Foam is open to contributions of any kind, including but not limited to code, do
 - Foam code and documentation live in the monorepo at [foambubble/foam](https://github.com/foambubble/foam/)
   - [/docs](https://github.com/foambubble/foam/docs): documentation and [[recipes]]  
   - [/packages/foam-vscode](https://github.com/foambubble/foam/tree/master/packages/foam-vscode): the core VSCode plugin
-  - [/packages/foam-workspace-manager](https://github.com/foambubble/foam/tree/master/packages/foam-workspace-manager): Foam workspace automations 
+  - [/packages/foam-workspace-manager](https://github.com/foambubble/foam/tree/master/packages/foam-workspace-manager): Foam workspace automations
 - Exceptions to the monorepo are:
   - The starter template at [foambubble/foam-template](https://github.com/foambubble/)
   - All other [[recommended-extensions]] live in their respective GitHub repos.
 
+## Contributing to the VS Code Extension
+
+If you're interested in contributing to the VS Code extension (aka `foam-vscode`), this guide will help you get things set up locally.
+
+1. Clone the repo locally:
+
+   `git clone https://github.com/foambubble/foam.git`
+2. Install the necessary dependencies by running this command from the root:
+
+   `yarn install`
+3. This project uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/).`foam-vscode` relies on `foam-workspace-manager`. This means we need to compile it before we do any extension development. From the root, run the command:
+
+   `yarn workspace foam-workspace-manager build`
+4. Now we'll use the launch configuration defined at [`.vscode/launch.json`](https://github.com/foambubble/foam/blob/master/.vscode/launch.json) to start a new extension host of VS Code. From the root, or the `foam-vscode` workspace, press f5.
+5. In the new extension host of VS Code that launched, open a Foam workspace (e.g. your personal one, or a test-specific one created from foam-template). This is strictly not necessary, but the extension won't auto-run unless it's in a workspace with a `.vscode/foam.json` file.
+6. Test a command to make sure it's working as expected. Open the Command Palette (Ctrl/Cmd + Shift + P) and select "Foam: Update Markdown Reference List". If you see no errors, it's good to go!
+
+For more resources related to the VS Code Extension, check out the links below:
+- [[Tutorial: Adding a New Command to the VS Code Extension]]
```

Comment:
Did I do this right @jevakallio ? It didn't get added to the autogenerated link references

Reply:
As you discovered in #68, and I explained further in [linked issue comment](https://github.com/foambubble/foam/issues/50#issuecomment-654500531) right now links with non-exact labels don't work yet.

This is the safe way for now:
```suggestion
- [[tutorial-adding-a-new-command-to-the-vs-code-extension]]
```

Alternatively, you can do this with a regular markdown link for the time being. It will work, but backlinks won't, as they're only supported for [[wiki-links]]:

```suggestion
- [Tutorial: Adding a New Command to the VS Code Extension](tutorial-adding-a-new-command-to-the-vs-code-extension.md)
```

Reply:
If you end up accepting the former suggestion, you still need to go and save the document in VS Code to generate link references. This is why we need [workspace janitor](https://foambubble.github.io/foam/workspace-janitor) sharpish!